WordPress/Classes/ViewRelated/Blog/QuickStartChecklistHeader.swift

```diff
@@ -15,11 +16,29 @@ class QuickStartChecklistHeader: UIView {
                 self?.bottomStroke.alpha = CGFloat(alpha)
                 self?.chevronView.transform = CGAffineTransform(rotationAngle: rotate)
             })
+
+            let hintToken: String
+            if collapse {
+                hintToken = NSLocalizedString("Collapses", comment: "This token is used with ~ to format the accessibility hint to reflect the expanded and collapsed state of the list.")
+            } else {
+                hintToken = NSLocalizedString("Expands", comment: "This token is used with ~ to format the accessibility hint to reflect the expanded and collapsed state of the list.")
+            }
+
+            let accessibilityFormat = NSLocalizedString("%@ the list of completed tasks.", comment: "Accessibility hint for the list of completed tasks presented during Quick Start.")
+            let localizedAccessibilityHint = String(format: accessibilityFormat, hintToken)
+
+            accessibilityHint = localizedAccessibilityHint
         }
     }
     var count: Int = 0 {
         didSet {
             titleLabel.text = String(format: Constant.title, count)
+
+            // Admittedly pluralization support could be improved here. See also: `paaHJt-c3-p2`
+            let accessibilityFormat = NSLocalizedString("%i completed tasks, toggling expands & contracts the list of these tasks", comment: "Accessibility description for the list of completed tasks presented during Quick Start.")
```

Comment:
Since this control holds a state, I think it would be good to state its current state in the accessibility label.
Something like: "Expanded. 1 Completed task. Button. (hint) expans & collapse the list of completed tasks"

Reply:
It also might be helpful to let translators now that `%i` is a number (the count of completed tasks).